src/server/services/decos/decos-service.ts

```diff
@@ -675,6 +686,58 @@ export async function fetchDecosDocument(
   );
 }
 
+export function transformDecosZaakFrontend<T extends DecosZaakBase>(
```

Comment:
Ik snap dat je deze in Decos zet, 9 van de 10 keer transformeren we zo'n decos zaak naar onderstaande. Voor de volledigheid zou je ipv de term `vergunning` hier ook `zaak` of `decosZaak` kunnen gebruiken.